rest-api/controllers/pokemon_feed.go

```diff
@@ -102,6 +106,81 @@ func Api_feed(w http.ResponseWriter, r *http.Request) {
 		json.NewEncoder(w).Encode(fs)
 	}
 }
+func worker_add(id int, jobs <-chan Pokemon, results chan<- Pokemon) {
```

Comment:
It works, but adding unit test coverage would be great

Reply:
unit test has been added too in my last commit